server/base/src/main/java/org/apache/accumulo/server/constraints/MetadataConstraints.java

```diff
@@ -390,20 +386,12 @@ private void validateServerFamily(ArrayList<Short> violations, ColumnUpdate colu
 
     switch (qualStr) {
       case ServerColumnFamily.LOCK_QUAL:
-        if (zooCache == null) {
-          zooCache = new ZooCache(context.getZooReader(), null);
-          CleanerUtil.zooCacheClearer(this, zooCache);
```

Comment:
may be able to remove this static method now.

Reply:
Removed the method.